### PR TITLE
Manually reconnect DxValidationSummary to editors (T1036547)

### PR DESCRIPTION
**Pull request by IgnatovDan**

### CODE REVIEW

File: js/ui/form/ui.form.js [hunk 1]
```diff
@@ -578,6 +578,9 @@ const Form = Widget.inherit({
                 const nestedItemsRunTimeInfo = component.getItemsRunTimeInfo();
                 this._itemsRunTimeInfo.removeItemsByItems(nestedItemsRunTimeInfo);
             },
+            onFieldItemRendered: () => {
+                this._validationSummary && this._validationSummary._initGroupRegistration();
```
Comment (ksercs) [inline js/ui/form/ui.form.js]:
```suggestion
                this._validationSummary?._initGroupRegistration();
```

File: js/ui/form/ui.form.layout_manager.js [hunk 2]
```diff
@@ -593,6 +593,8 @@ const LayoutManager = Widget.inherit({
             labelMode: this.option('labelMode'),
         }));
 
+        this.option('onFieldItemRendered') && this.option('onFieldItemRendered')();
```
Comment (ksercs) [inline js/ui/form/ui.form.layout_manager.js]:
```suggestion
       this.option('onFieldItemRendered')?.();
```

**Review by ksercs — DISMISSED** [review]

File: js/ui/form/ui.form.js (hunk 1 above)
Comment (IgnatovDan) [reply js/ui/form/ui.form.js]: Done

Comment (IgnatovDan) [reply js/ui/form/ui.form.js]: Done

File: js/ui/form/ui.form.layout_manager.js (hunk 2 above)
Comment (IgnatovDan) [reply js/ui/form/ui.form.layout_manager.js]: Done

**Review by novsstation — APPROVED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Pull request merged**